src/components/BulkActionsToolbar/BulkActionsToolbar.tsx

```diff
@@ -64,15 +67,15 @@ export const BulkActionsToolbar: React.FC<Props> = (props) => {
                 </Tooltip>
             </div>
             <div {...elem('actions')}>
-                {actions.map((action) => {
+                {actions.map((action, index) => {
                     // We have to wrap the button into <span/> in order to
                     // be able to show the tooltip on disabled button.
                     // https://github.com/atomiks/tippyjs-react/issues/123
                     return (
                         <Tooltip key={action.label} content={action.tooltip}>
-                            <span {...elem('actionWrapper')}>
+                            <span {...elem('actionWrapper', { first: index === 0 })}>
```

Comment:
Is this wrapper around each button necessary? Can't that left margin be applied directly to the buttons?

Reply:
Yes, span is necessary, there is comment:

> // We have to wrap the button into <span/> in order to
                    // be able to show the tooltip on disabled button.
                    // https://github.com/atomiks/tippyjs-react/issues/123

but since all main button styles are added to button - I've moved margin there as well